package/utils/omnia-eeprom/Makefile

```
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2024 Marek Behún <kabel@kernel.org>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=omnia-eeprom
+PKG_VERSION:=v0.1
+PKG_RELEASE:=$(AUTORELEASE)
```

`AUTORELEASE` has been deprecated and should not be used.

---

Yeah, easy to fix to apply this suggestion:
```suggestion
PKG_RELEASE:=1
```